tools/format-bazel-files.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Formats and fixes all Bazel files (*.bazel, *.bzl) using Buildifier.
+bazel run -- @buildifier_prebuilt//:buildifier \
```

Comment:
Does this command work whereever you call it? Do you need to set `bazel_info_workspace`?

Reply:
The Bazel workspace root is the "closest" folder containing a `WORKSPACE.bazel` or `MODULE.bazel` file. Since these files sit in the p4c root directory, this script will work as long as we call it from any subfolder within the p4c project.

Reply:
I guess my confusions is around the variable `${bazel_info_workspace}`. Who sets this? Do you need to do this manually?

Reply:
Yes! Seems like I need to fix my eyesight. :)